.github/workflows/nist_data.yml

```diff
@@ -0,0 +1,91 @@
+name: Check NIST ionization energies and atomic weights
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'  #runs on every Sunday midnight
+  workflow_dispatch:
+  
+env:
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
+
+jobs:
+  build:
+    
+    defaults:
+     run:
+       shell: bash -l {0}
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: act10ns/slack@v2.0.0
+        with:
+          status: starting
+          message: Starting to check NIST data
+        if: always()
+        
+      - name: Checkout github repo
+        uses: actions/checkout@v3
+
+      - name: Configure git
+        run: |
+          git config --global user.name "GitHub Actions"
+          git config --global user.email "actions@github.com"
+
+      - name: Clone tardis-sn/carsus
+        uses: actions/checkout@v3
+        with:
+          repository: tardis-sn/carsus
+          path: carsus
+
+      - name: Setup environment
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+            miniforge-variant: Mambaforge
+            miniforge-version: latest
+            environment-file: carsus/carsus_env3.yml
+            activate-environment: carsus
+            use-mamba: true
+
+      - name: Install carsus
+        id: install_carsus
+        run: pip install ./carsus
+                     
+      - name: Run Python ionization energies script
+        id: run_ionization_energies_script
+        run: | 
+          python ionization.py
+
+      - name: Run Python atomic weights script 
+        id: run_atomic_weights_script
+        run: | 
+          python atomic_weights.py
+          
+      - name: Compare changes
+        id: compare
+        continue-on-error: true
+        run: |
+          git add .
+          git rm --cached -f carsus
```

Comment:
I don't think we should use the -f option, did the `git add abc xyz not work`?

Reply:
We have to use `-f` and do force removal, otherwise it's not removed.

Reply:
Ohh! I was wrong. It's working. I have modify the code